web/src/app/selection/MaterialSelect.js

```diff
@@ -1,10 +1,152 @@
-import React, { Component } from 'react'
+import React, { useState } from 'react'
 import { PropTypes as p } from 'prop-types'
-import { withStyles } from '@material-ui/core/styles'
-import Select from 'react-select'
-import { components, styles } from './MaterialSelectComponents'
-import shrinkWorkaround from '../util/shrinkWorkaround'
-import _ from 'lodash-es'
+import {
+  TextField,
+  makeStyles,
+  MenuItem,
+  ListItemIcon,
+  Typography,
+  Paper,
+  Chip,
+} from '@material-ui/core'
+import { Autocomplete } from '@material-ui/lab'
+
+export const styles = () => ({
```

Comment:
I don't see this being imported anywhere, so we should be fine leaving the `export` out and removing the constant in favor of defining the styles within the func's parameter